src/libs/ui/src/organisms/Product/index.tsx

```diff
@@ -1,63 +1,71 @@
-
+import { Link } from 'react-router-dom';
 import { Button } from '@ui/atoms/Button';
 import { ProductImage } from '@ui/atoms/ProductImage';
 import StarRating from '@ui/atoms/StarRating';
 import BestSellerBadge from '@ui/molecules/BestSeller';
 import QuickViewButton from '@ui/molecules/QuickViewButton';
 import { ProductProps } from '@utils/Product';
 
+function Product({ product, modalSetToggle, className, openQuickView, showQuickView }: ProductProps) {
+  const { image, images, name, isBestSeller, description, features, rating, id } = product;
 
-function Product({ product ,modalSetToggle,className, openQuickView,showQuickView }: ProductProps) {
-  
-  const { image, name, isBestSeller, rating ,id} = product;
-
-
- 
   return (
-   
-    <div className={'group relative p-2 '}>
-      <div>
-        <ProductImage src={image} alt={name} className='h-[305px]'></ProductImage>
+    <Link
```

Comment:
Please remove your changes from this Product component,  please discuss with @Nishant8209  to add click event to this Component

Reply:
Made changes as requested